addons/l10n_ec/data/account_tax_template_withhold_profit_data.xml

```diff
@@ -0,0 +1,7906 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data >
```

Comment:
This file is huge and only applied on salary, profits, ...  Maybe it is better to leave it out at this point.